dashboard/app/controllers/pd/workshop_daily_survey_controller.rb

```diff
@@ -449,6 +449,11 @@ def render_csf_survey(survey_name, workshop)
 
     def render_csf_survey_foorm(survey_name, workshop)
       form_questions, latest_version = ::Foorm::Form.get_questions_and_latest_version_for_name(survey_name)
+      facilitators = workshop.facilitators
+      facilitator_data = []
```

Comment:
You can use `map`, it's more convenient.
```suggestion
facilitator_data = workshop.facilitators.map do |facilitator|
  {facilitatorId: facilitator.id, facilitatorName: facilitator.name}
end
```